components/director/internal/domain/formationtemplate/resolver.go

```diff
@@ -280,3 +298,47 @@ func (r *Resolver) Webhooks(ctx context.Context, obj *graphql.FormationTemplate)
 
 	return gqlWebhooks, nil
 }
+
+// FormationConstraint retrieves a FormationConstraint for the specified FormationTemplate
+func (r *Resolver) FormationConstraint(ctx context.Context, obj *graphql.FormationTemplate) ([]*graphql.FormationConstraint, error) {
+	params := dataloader.ParamFormationConstraint{ID: obj.ID, Ctx: ctx}
+	return dataloader.FormationTemplateFor(ctx).FormationConstraintByID.Load(params)
+}
+
+// FormationConstraintsDataLoader retrieves Formation Constraints for each FormationTemplate ID in the keys
+func (r *Resolver) FormationConstraintsDataLoader(keys []dataloader.ParamFormationConstraint) ([][]*graphql.FormationConstraint, []error) {
```

Comment:
As we discussed, if we agree to add Page structures it will be done in a follow-up PR.

To sum up the discussion: 
Currently, we don't need the Page structures for the pagination, because the Formation Constraints are just a few. But the Page structures could make the code more readable because we will not work with an array of arrays([][]...) but with an array of page structures. If that is the case, it will be better to add pages.